src/auth/src/user_provider/static_user_provider.rs

```diff
@@ -60,6 +60,18 @@ impl StaticUserProvider {
                 .fail(),
         }
     }
+
+    /// Make sure there is only one username/password pair
+    /// And return it
+    pub fn get_one_user_pwd(&self) -> Result<(String, String)> {
+        let kv = self.users.iter().next().context(InvalidConfigSnafu {
+            value: "",
+            msg: "Expect one and only one pair of username and password",
```

Comment:
If there are multiple auth pairs, we can take one of them. 
I assume that the config would be the same for both frontend and flownode for convenience. In that case the frontend can have multiple auth pairs, where the flownode can take one valid pair for later usage.

Reply:
> If there are multiple auth pairs, we can take one of them. I assume that the config would be the same for both frontend and flownode for convenience. In that case the frontend can have multiple auth pairs, where the flownode can take one valid pair for later usage.

is take one randomly acceptable? since the underline one is hash map

Reply:
I think it is acceptable. The goal here is to take a pair of valid ones, it doesn't seem matter to me if it's the first pair or the last. 
Also, we can add some comments to state the usage of this function (to enable invoking other components within the cluster), otherwise the function itself seems strange.